docx/oxml/numbering.py

```diff
@@ -192,7 +192,7 @@ def get_preceding_paragraphs_numIds(p, p_ilvl, p_numId):
                     if prev_p_ilvl < p_ilvl and (prev_p_numId == p_numId or
```

Comment:
While testing this I've also put the same equation in this if check. Everything worked. We should discuss it probably.